otherlibs/unix/getgr.c

```diff
@@ -28,7 +28,7 @@ static value alloc_group_entry(struct group *entry)
 
   Begin_roots3 (name, pass, mem);
     name = caml_copy_string(entry->gr_name);
-    pass = caml_copy_string(entry->gr_passwd);
+    pass = caml_copy_string(entry->gr_passwd ? entry->gr_passwd : "");
```

Comment:
gr_passwd is always NULL, blame android :] Cannot change the signature, so this workaround to not segfault...
Witness : 
```
./gu0_a212@localhost:~/tmp$ gcc gr.c -o gr
./gu0_a212@localhost:~/tmp$ ./gr
gr_name root gr_passwd (null) gr_gid 0 
u0_a212@localhost:~/tmp$ cat gr.c
#include <sys/types.h>
#include <unistd.h>
#include <grp.h>
#include <stdio.h>

int main()
{
  struct group* p = getgrgid(0);
  printf("gr_name %s gr_passwd %s gr_gid %d \n", p->gr_name, p->gr_passwd, p->gr_gid);
  return 0;
}
```